dev-libs/rocksdb/rocksdb-6.17.3.ebuild

```diff
@@ -27,6 +27,8 @@ RDEPEND="${DEPEND}"
 
 PATCHES=(
 	"${FILESDIR}"/${PN}-6.14.6-libatomic.patch
+	"${FILESDIR}"/${PN}-6.17.3-add_timer_for_riscv.patch
```

Comment:
I usually will squash the first two commits (adding patches) along with "PATCHES" change in the ebuild, but since we're using git, so I'm personally fine with what it is here (no breakage).  please note, as jsmolic suggested, we usually have keyword changed in separate commit.

Reply:
Gotcha, thanks for the suggestion!

Reply:
well, if you really want to squash the changes, then I'd sugest move "*-add_timer_for_riscv.patch" to previous commit: "dev-libs/rocksdb: add timer for riscv platform"

it's quite fair, you add the patch and also adjust PATCHES, so it's atomic (if we have to revert it later just in case, then it's will be clean revert)